File: app/src/main/java/com/enipro/data/remote/model/MessageUser.java
package com.enipro.data.remote.model;


import com.stfalcon.chatkit.commons.models.IUser;

public class MessageUser implements IUser {

    private String name;
    private String id;
    private String avatar;

    public MessageUser(String id, String name, String avatar) {
        this.id = id;
        this.name = name;
        this.avatar = avatar;
    }


    @Override
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    @Override
    public String getAvatar() {
        return avatar;
    }

    public void setAvatar(String avatar) {
        this.avatar = avatar;
    }
}

File: app/src/main/java/com/enipro/data/remote/model/MessageUser.kt
package com.enipro.data.remote.model


import com.stfalcon.chatkit.commons.models.IUser

data class MessageUser(private var id: String?, private var name: String?, private var avatar: String?) : IUser {


    override fun getName(): String? {
        return name
    }

    override fun getId(): String? {
        return id
    }

    override fun getAvatar(): String? {
        return avatar
    }

}
